realm/realm-library/src/syncIntegrationTest/java/io/realm/objectserver/EncryptedSynchronizedRealmTests.java

```diff
@@ -146,7 +143,7 @@ public void onError(SyncSession session, ObjectServerError error) {
         try {
             realm = Realm.getInstance(configWithoutEncryption);
             fail("It should not be possible to open the Realm without the encryption key set previously.");
-        } catch (RealmFileException ignored) {
+        } catch (RealmError ignored) {
```

Comment:
This feels like a regression? What this change?

Reply:
an `std::exception` is thrown (`Invalid mnemonic. top_ref[0]:`) which doesn't map to the `RealmFileException` anymore ...

Reply:
What triggered the mapping in the past? Sounds like something we would want to fix in Core?

Reply:
This looks like an oversight in OS, see https://mongodb.slack.com/archives/CKW2HG3M0/p1576101304053900